invenio_app_ils/document_requests/loaders/jsonschemas/document_request.py

```diff
@@ -56,9 +56,12 @@ class Meta:
     note = SanitizedHTML()
     page = SanitizedHTML()
     patron_pid = fields.Str(required=True, validate=validate_patron)
+    payment_info = fields.Str()
```

Comment:
Food for thought, I am not sure if I would go for the explicit `payment_*` fields or if I would create a complex field `payment` with `info` and `method` as properties, very similar to `price`, @kprzerwa, @ntarocco

Reply:
yes maybe consult this as a team